testing_and_setup/compass/utility_scripts/LIGHTparticles/make_particle_file.py

```diff
@@ -579,11 +630,18 @@ def build_particle_file(f_init, f_name, f_decomp, types, spatialfilter, buoySurf
 
     assert set(args.types.split(',')).issubset(typelist), 'Selected particle type is not correct!'
 
+    # Defaults to center seeding for particles.
+    if not args.seed_center and not args.seed_vertex:
+        args.seed_center = True
+
+    if args.add_noise and not args.seed_center:
+        raise ValueError('Gaussian noise requested but center-seeding not requested. Please resubmit this function with `--center` and `--add_noise`.')
```

Comment:
Please split into multiple lines.

Reply:
Done. Will convert this to python3 down the line and will run `black` on this for syntax.